packages/toolkit/package.json

```diff
@@ -52,8 +52,4185 @@
     ],
     "main": "./dist/src/main",
     "browser": "./dist/src/mainWeb",
-    "engines": "This field will be autopopulated from the core module during debugging and packaging.",
-    "contributes": "This field will be autopopulated from the core module during debugging and packaging.",
+    "engines": {
```

Comment:
Question: This section is "re-generated" because of above `package.json` change?

Reply:
No, I'm not sure why it's regenerated, but does not have to do with my `package.json` changes.

Reply:
I will remove this `package.json` from this PR though.